### ISSUE TITLE
OneUpTime SSO with Azure Entra Id user syncing

### ISSUE THREAD

**Issue by KunduAnilReddy** [OneUptime/oneuptime#1711]
**Describe the bug**
I have configured the OneUpTime SSO with Azure Entra ID but the users are not syncing automatically when the user logging into the OneUpTime using SSO .

**To Reproduce**
1.I have configured the OneUpTime SSO with Azure Entra ID but the users are not syncing automatically when the user logging into the OneUpTime using SSO .
2.User only can able to login using SSO when I add them into the oneuptime team members otherwise SSO automatically not syncing from AzureAD to oneuptime with SSO implementation.

**Expected behavior**
User Should be able to login using SSO automatically once I add them into the Azure AD group

**Deployment Type**
 self hosted 

**Additional context**
1.Can SomeOne confirm Whether  users will automatically sync up once we implement OneUpTime SSO with AzureAD or we Need to add them into the OneUpTime portal also to get access using SSO ?

2. Also can someone suggest is there anyway to syncup the users automatically from azuread to oneuptime? example : Oauth ,etc....

**Comment by KunduAnilReddy** [comment]
Hi simlarsen : 
Thanks for the reply . I have followed this video (https://www.youtube.com/watch?v=9TN0my4OENI) to implement the SSO oneuptime with my enterprise application . Whatever they have mentioned in the video I Have followed same steps and it is working when I invite users both sides enterprise application and one uptime portal , So it is not syncing the user automatically once I invite the user in azure enterprise application. Can you suggest what is the issue here or suggest if we have any other better approach to achieve this SSO auto syncing? 
Thank you !

**Comment by simlarsen** [comment]
Users should be automatically created on OneUptime when they log in for the first time from Azure Entra ID. Is that not the case?

**Comment by KunduAnilReddy** [comment]
Yes , That is the problem in our case user can't able to sync automatically from Azure Entra ID ? When I add  the user in the oneuptime and azure entra id then only user can able to login into the oneuptime . Not sure what is the issue

**Comment by KunduAnilReddy** [comment]
Hi @simlarsen  - Any update on my issue?

**Comment by simlarsen** [comment]
We currently do not sync users, so they wont appear until they sign up. 

Please let me know if this helps!

**Comment by simlarsen** [comment]
This is complete with SCIM. This will be released by EOD today.
